modules/platforms/dotnet/Apache.Ignite.Core/Binary/BinaryReflectiveSerializer.cs

```diff
@@ -46,6 +46,7 @@ public sealed class BinaryReflectiveSerializer : IBinarySerializer
         private bool _isInUse;
 
         /** Force timestamp flag. */
+        [Obsolete("Deprecated, use BinaryConfiguration.ForceTimestamp instead.")]
```

Comment:
I don't think this deprecation is necessary. This flag is still useful.

PS `[Obsolete]` should be applied to a public property, not a private field

Reply:
Reverted.